UniMath/CategoryTheory/.package/files

```diff
@@ -18,13 +18,13 @@ limits/initial.v
 limits/graphs/initial.v
 limits/zero.v
 limits/graphs/zero.v
+limits/bincoproducts.v
 limits/coproducts.v
-limits/arbitrary_coproducts.v
-limits/graphs/coproducts.v
+limits/graphs/bincoproducts.v
```

Comment:
I prefer all lowercase letters in filenames, but I'm not too fanatic about it (it is not like trailing whitespace... :)). However there seems to be no standard for this in UniMath at the moment. Maybe we could decide on a standard for the CategoryTheory library and make another PR where we make the file names more uniform?